test/problem1_spec.rb

```diff
@@ -0,0 +1,37 @@
+
+require 'rspec'
+require '../src/problem1'
+
+describe Problem1 do
+  describe '.sum_of_even_numbers' do
```

Comment:
When describing method calls, you can use `subject` to avoid repeating whole method call.

Reply:
I read something, but I have been running out of time :(